scst-policy/configuring.md

```diff
@@ -1,205 +1,142 @@
+apiVersion: cosigned.sigstore.dev/v1alpha1
 kind: ClusterImagePolicy
 metadata:
   name: image-policy
 spec:
-  verification:
-    exclude:
-      resources:
-        namespaces:
-        - kube-system
-        - accelerator-system
-        - api-portal
-        - app-live-view
-        - app-live-view-connector
-        - app-live-view-conventions
-        - build-service
-        - cartographer-system
-        - cert-injection-webhook
-        - cert-manager
-        - conventions-system
-        - developer-conventions
-        - flux-system
-        - image-policy-system
-        - kapp-controller
-        - knative-eventing
-        - knative-serving
-        - knative-sources
-        - kpack
-        - learning-center-guided-ui
-        - learning-center-guided-w01
-        - learningcenter
-        - metadata-store
-        - scan-link-system
-        - secretgen-controller
-        - service-bindings
-        - services-toolkit
-        - source-system
-        - spring-boot-convention
-        - stacks-operator-system
-        - tanzu-cluster-essentials
-        - tanzu-package-repo-global
-        - tanzu-system-ingress
-        - tap-gui
-        - tap-install
-        - tap-telemetry
-        - tekton-pipelines
-        - triggermesh
-    keys:
-    - name: cosign-key
-      publicKey: |
-        -----BEGIN PUBLIC KEY-----
-        MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhyQCx0E9wQWSFI9ULGwy3BuRklnt
-        IqozONbbdbqz11hlRJy9c7SG+hdcFl9jE9uE/dwtuwU2MqU9T/cN0YkWww==
-        -----END PUBLIC KEY-----
+  images:
+  - glob: "*"
+```
+
+### <a id="cip-authorities"></a> `authorities`
+
+Authorities listed in the `authorities` block of the ClusterImagePolicy may be
+`key` specifications. Each `key` authority may contain a raw public key, a
+`secretRef` or a `kms` path.
+
+**Note**: Currently, only ECDSA public keys are supported.
+
+```yaml
+spec:
+  authorities:
+    - key:
+        data: |
+          -----BEGIN PUBLIC KEY-----
+          ...
+          -----END PUBLIC KEY-----
+    - key:
+        secretRef:
+          name: secretName
+    - key:
+        kms: KMSPATH
+```
+
+Authorities may be `keyless` specifications. Each keyless authority may contain
+a Fulcio URL, a certificate or an array of identities.
+
+```yaml
+spec:
+  authorities:
+    - keyless:
+        url: https://fulcio.example.com
+        ca-cert:
+          data: Certificate Data
+    - keyless:
+        url: https://fulcio.example.com
+        ca-cert:
+          secretRef:
+            name: secretName
+    - keyless:
+        identities:
+          - issuer: https://accounts.google.com
+            subject: .*@example.com
+          - issuer: https://token.actions.githubusercontent.com
+            subject: https://github.com/mycompany/*/.github/workflows/*@*
+```
+
+The following is an example `ClusterImagePolicy` which matches against all
+images using a glob:
+
+```yaml
+apiVersion: cosigned.sigstore.dev/v1alpha1
+  kind: ClusterImagePolicy
```

Comment:
this shouldnt be tabbed in (and below) and should be on the same root node level as `apiVersion`